.github/workflows/weekly-preview.yml

```diff
@@ -13,10 +13,10 @@ jobs:
       with:
         ref: dev
         fetch-depth: 0
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
May I ask why we need to update to 3.9 here?
Do we also update here?
https://github.com/Project-MONAI/MONAI/blob/865972f7a791bf7b42efbcd87c8402bd865b329e/setup.cfg#L36

Reply:
This is for dev preview, at some point next year (1.4 or 1.5) we'll remove the 3.8 support..